code/modules/research/designs/bot_upgrades.dm

```diff
@@ -0,0 +1,43 @@
+///////////////////
+// RESEARCH ITEM //
+///////////////////
+
+/datum/design/bot_upgrade
+	// Should not be acquirable
+
+/datum/design/bot_upgrade/boost
+	name = "Bot Upgrade: Boost"
+	desc = "The circuit board for giving a speed boost to bots"
+	id = "bot_upgrade_boost"
+	req_tech = list("programming" = 2, "engineering" = 2)
+	build_type = MECHFAB
+	materials = list(MAT_GLASS = 500)
+	build_path = /obj/item/weapon/bot_upgrade/boost
+	category = list("Misc")
+	construction_time = 100
+
+////////////////////////
+// CIRCUIT BOARD ITEM //
+////////////////////////
+
+/obj/item/weapon/bot_upgrade/
+	density = 0
+	anchored = 0
+	w_class = 2
+	name = "bot upgrade"
+	icon = 'icons/obj/module.dmi'
+	icon_state = "id_mod"
+	item_state = "electronic"
+	origin_tech = "programming=2"
+	materials = list(MAT_GLASS=500)
+	var/board_type = "computer"
```

Comment:
More copy paste (read: bad habit) I believe this variable is just verification for when you're building computers, so this actually needs to be removed